eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -354,51 +353,6 @@ void choosesTransformerFactoryInConcurrentEnvironment() {
         }
     }
 
-    @Test
-    void failsOnErrorAfterNotReplacingWrongVersions(@TempDir final Path tmp) {
```

Comment:
@maxonfjvipon Maybe we can leave checking of versions?

Reply:
@volodya-lombrozo see [this comment](https://github.com/objectionary/eo/pull/2386#discussion_r1293344733). Since we don't check versions via `xsl` there's no point of the test

Reply:
@maxonfjvipon I understood. Thank you.